go/vt/grpcclient/snappy.go

```diff
@@ -44,17 +39,3 @@ func (s SnappyCompressor) Compress(w io.Writer) (io.WriteCloser, error) {
 func (s SnappyCompressor) Decompress(r io.Reader) (io.Reader, error) {
 	return snappy.NewReader(r), nil
 }
-
```

Comment:
GH won't let me comment on the actual interface implementation, but: see my comments about the doc comments for the Zstd implementation; let's clean these up similarly while we're in here.